src/collections.ts

```diff
@@ -29,7 +29,7 @@ registry.register(
       'deprecated-css-custom-property-syntax',
       'deprecated-shadow-dom-selectors',
       'dom-calls-to-native',
-      'dom-module-invalid-attrs',
+      'dom-module-name-or-is',
```

Comment:
We need to change the file names, as changing a rule id is a breaking user-visible change.

Reply:
Oh, I'm sorry. I misunderstood what you were wanting. Will update that here soon.

Reply:
np!